pom.xml

```diff
@@ -22,108 +22,44 @@
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Git+Plugin</url>
 
   <properties>
-    <findbugs-maven-plugin.version>3.0.3</findbugs-maven-plugin.version>
-    <jacoco-maven-plugin.version>0.7.4.201502262128</jacoco-maven-plugin.version>
-    <maven-compiler-plugin.version>3.5.1</maven-compiler-plugin.version>
-    <maven-enforcer-plugin.version>1.4.1</maven-enforcer-plugin.version>
-    <maven-hpi-plugin.version>1.117</maven-hpi-plugin.version>
-    <maven-localizer-plugin.version>1.23</maven-localizer-plugin.version>
-    <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
-    <maven-site-plugin.version>3.5</maven-site-plugin.version>
-    <maven-surefire-plugin.version>2.19.1</maven-surefire-plugin.version>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <jenkins.version>1.609</jenkins.version>
```

Comment:
Please be aware that the baseline change is a no-op from a deployment perspective: the plugin has been effectively requiring 1.609 since https://github.com/jenkinsci/git-plugin/commit/8cec686d1d526a1fa4b2575fcd9e6711d9b7f6e9.

Regarding the upgrade to Java 7 and 1.625, I agree with your approach @MarkEWaite, and I would go for that in a downstream PR.